src/core/worker.js

```diff
@@ -436,6 +442,55 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
       });
     });
 
+    handler.on('GetRawObject', function wphGetRawObject(data) {
+      function clone(obj) {
+        if (isBool(obj) || isNum(obj) ||
+            isString(obj) || isNull(obj)) {
```

Comment:
There's no longer a function called `isNull`, so this line will need to change. I suggest the following instead (and note that it should now fit on *one* line):
```javascript
  if (!obj || isBool(obj) || isNum(obj) || isString(obj)) {
```
Also, you should remove `isNull` from the list of `globals` at the top of the file.